MediaBrowser.Providers/Music/MusicBrainzAlbumProvider.cs

```diff
@@ -14,24 +15,13 @@
 using MediaBrowser.Controller.Providers;
 using MediaBrowser.Model.Entities;
 using MediaBrowser.Model.Providers;
-using MediaBrowser.Model.Serialization;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 
 namespace MediaBrowser.Providers.Music
 {
     public class MusicBrainzAlbumProvider : IRemoteMetadataProvider<MusicAlbum, AlbumInfo>, IHasOrder
     {
-        internal static MusicBrainzAlbumProvider Current;
-
-        private readonly IHttpClient _httpClient;
-        private readonly IApplicationHost _appHost;
-        private readonly ILogger _logger;
-        private readonly IJsonSerializer _json;
-        private Stopwatch _stopWatchMusicBrainz = new Stopwatch();
-
-        public readonly string MusicBrainzBaseUrl;
```

Comment:
Something in this file is breaking the syntax highlighting, any clue why?

Reply:
No, looks fine